ofl/bricolagegrotesque/METADATA.pb

```diff
@@ -34,5 +34,5 @@ axes {
 }
 source {
   repository_url: "https://github.com/ateliertriay/bricolage"
-  commit: "95ddb0da9087a5f49f4616faea519e9e27d80a99"
+  commit: "c3739c1d4ee94e1b3c130453b74a52a1cb5062b0"
 }
```

Comment:
Can add `minisite_url: https://ateliertriay.github.io/bricolage/` to the metadata file.  Mathieu confirmed that this will be the prod URL for the minisite